lib/build-config.js

```diff
@@ -78,8 +78,7 @@ async function removeBuildsAndBuildConfig (config) {
 async function createOrUpdateBuildConfig (config) {
   // Check for a BuildConfig, create or update if necessary
   const buildName = config.buildName;
-  const imageStreamName = config.projectName;
-  const outputImageStreamTag = `${imageStreamName}:latest`; // TODO: base on a tag
+  const outputImageStreamTag = `${config.outputImageStreamName}:latest`; // TODO: base on a tag
```

Comment:
Shouldn't you be using the `outputImageSreamTag` option here instead of `latest`?

Reply:
yea,  probably :)

Reply:
Updated with the new commits